src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -76,12 +84,17 @@ def eval(**kwargs):
 
 
 def oneshot(**kwargs):
+    from llmcompressor.transformers.calibration.oneshot import Oneshot
+
     """
     CLI entrypoint for running oneshot calibration
     """
-    model_args, data_args, training_args = parse_args(**kwargs)
-    training_args.do_oneshot = True
-    main(model_args, data_args, training_args)
+    oneshot_calibrator = Oneshot(**kwargs)
```

Comment:
Why are we calling this a calibrator?

Reply:
You are right, its running ptq and compressing. Ill call it compressor/oneshot_compressor